python/cugraph-service/client/cugraph_service_client/remote_graph.py

```diff
@@ -369,15 +423,40 @@ def get_edge_data(
         edge_ids=None,
         types=None,
         columns=None,
-        backend=("cudf" if cudf is not None else "numpy"),
+        backend=("cudf" if not isinstance(cudf, MissingModule) else "numpy"),
```

Comment:
not critical at all, but would it be easier to define `cudf_installed = not(isinstance(cudf, MissingModule))` somewhere near the imports, then change code like this to:
```suggestion
        backend=("cudf" if cudf_installed else "numpy"),
```

Reply:
done